src/meta/api/src/schema_api_impl.rs

```diff
@@ -1926,47 +1930,14 @@ impl<KV: kvapi::KVApi<Error = MetaError>> SchemaApi for KV {
                 "upsert_table_copied_file_info"
             );
 
-            let mut condition = vec![txn_cond_seq(&tbid, Eq, tb_meta_seq)];
```

Comment:
upsert_table_copied_file_info in trait Catalog can be removed?

Reply:
yeah, seems no other components using this method now, let me try rm it

Reply:
if we need to keep it,  maybe we can add a update to table version in txn?